domains/moonzdevelopment.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "hymoonz",
+        "email": "bilal190620@gmail.com"
+    },
+    "record": {
+        "TXT": "replit-verify=8cf3fb31-5df3-4217-b863-7d4088af08ec"
```

Comment:
TXT record alone doesn't do anything.

Reply:
done sir

Reply:
No changes were made to the file though...?

Reply:
No sir

Reply:
That's just the beginning to create my profile

Reply:
I've changed it